src/main/java/org/folio/circulation/resources/RegularCheckOutStrategy.java

```diff
@@ -33,6 +33,7 @@
   public CompletableFuture<Result<LoanAndRelatedRecords>> checkOut(LoanAndRelatedRecords relatedRecords,
                                                                    JsonObject request,
                                                                    Clients clients) {
+
```

Comment:
Why do you need this line?

Reply:
Need? I didn't even notice this line.
Probably the result of my recent commit reverts (see the commit log of this PR).
Easy enough to remove.